### ISSUE TITLE
Fields 

### ISSUE THREAD

**Issue by jngaravitoc** [EXP-code/EXP-docs#7]
It would be useful to have a definition of the fields returned by` pyEXP.field.FieldGenerator.slices` 
I always confused myself as to what d1 and dd are referring to. This could also be added to this part of the docs:
https://exp-docs.readthedocs.io/en/latest/topics/visualizing-fields.html?

**Comment by The9Cat** [comment]
Ah yes.  Thanks for reminding me.

One suggestion from earlier is to make the dictionary keys themselves more descriptive, too.  E.g.

- `density` instead of `d`
- `density(m=0)` instead of `d0`
- `density(m>0)` instead of `d1`
- `relative density` instead of `dd`
- `phi force` or `force_phi` instead of `fp`
- `radial force` or `force_r` instead of `fr`
- `tangential force` or `force_tan` instead of `ft`
- `potential` instead of `p`
- `potential(m=0)` instead of `p0`
- `potential(m>0)` instead of `p1`

My main concern is that this keys are longer to type.  Obviously, I could write `dens` instead of `density` and `potl` instead of `potential` for some shortening of keys.  Thoughts?  When we have a consensus, I'll change the keys and the documentation.

**Comment by jngaravitoc** [comment]
Thank you! I think that adding that description to doc-strings in the functions would be sufficient

**Comment by The9Cat** [comment]
Here's a revised proposal for dictionary keys:

- `dens` is the total density
- `dens m=0` is the axisymmetric component of the density
- `delta dens` is the asymmetric component of the density.  This could be dropped since it's computable from the first two.
- `rel dens` is the ratio of the asymmetric to axisymmtric density.  This could be dropped also since it is computable from the first two density fields.
- `azi force` is the azimuthal force
- `rad force` is the radial force
- `mer force` is the meridional force
- `potl` is the gravitational potential
- `potl m=0` is the axisymetric component of the gravitational potential
- `delta potl` is the asymmetric component of the potential.  This could be dropped and let the user compute this from the two above.

**Comment by The9Cat** [comment]
This addressed by commit EXP-code/EXP@1b952b5 .  This still needs testing however and changes in pyEXP-examples.

**Comment by michael-petersen** [comment]
I think the proposed dictionary keys are good. It would be best to edit `EXP-docs` in tandem with these changes so that users. Are we still able to port the docstrings into `EXP-docs` in a straightforward way?

**Comment by The9Cat** [comment]
Agreed.  I don't know how to automatically port the docstrings into `EXP-docs`.  I do recall that there might be a tool that turns them into Sphinx...

**Comment by The9Cat** [comment]
Here is the documentation for the Sphinx `autodoc` module: https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html

TL;DR: like doxygen, the `autodoc` module uses stylized decoration to parse Python docstrings for Sphinx generation.  I'd put this on some ToDo list, but not something I'm keen to take on.
